::: lineapy.exceptions.db_exceptions
